### PR TITLE
Fix AB export and deduplicate available bidders

### PR DESCRIPTION
**Pull request by mjoyce91**

### CODE REVIEW

File: talentmap_api/bureau/services/available_bidders.py [hunk 1]
```diff
@@ -66,12 +66,17 @@ def get_available_bidders_csv(request):
         cdo_name = f'{pydash.get(record, "cdo.last_name")}, {pydash.get(record, "cdo.first_name")}'
 
         fields = formatCSV(record, fields_info)
+
+        try:
+            ted = maya.parse(fields["ted"]).datetime().strftime('%m/%d/%Y')
+        except:
+            ted = 'None listed'
```
Comment (elizabeth-jimenez) [inline talentmap_api/bureau/services/available_bidders.py:73]: 😭  we can get rid of `ted` on line 51 then

Comment (mjoyce91) [reply talentmap_api/bureau/services/available_bidders.py:73]: I think we still need it? Because we get it from `fields`

Comment (elizabeth-jimenez) [reply talentmap_api/bureau/services/available_bidders.py:73]: ooohhh yeaaaa 🎊  woot

**Review by scott062 — APPROVED** [review]
Looks identical to the changes we made except your case change which doesn't affect it though 👍

**Pull request merged**